Add container apps hosting option (#1896)
dapr/docs: daprdocs/content/en/operations/hosting/azure-container-apps/index.md

---
type: docs
title: "Run Dapr in Azure Container Apps"
linkTitle: "Azure Container Apps"
weight: 3000
description: "Learn how to run your Dapr apps on the Azure Container Apps serverless runtime"
---

> [Azure Container Apps](https://docs.microsoft.com/en-us/azure/container-apps/overview) is a serverless application centric hosting service where users do not see or manage any underlying VMs, orchestrators, or other cloud infrastructure. Azure Container Apps enables executing application code packaged in any container and is unopinionated about runtime or programming model.

Reviewer: ```suggestion
[Azure Container Apps](https://docs.microsoft.com/en-us/azure/container-apps/overview) is a serverless application centric hosting service where users do not see nor manage any underlying VMs, orchestrators, or other cloud infrastructure. Azure Container Apps enables executing application code packaged in any container and is unopinionated about runtime or programming model.

```

Dapr is built-in as a first-class experience in Container Apps, letting you use Dapr building blocks without any manual deployment or management of Kubernetes or the Dapr runtime. Simply deploy your services and Dapr components and let the platform take care of the rest.

{{< button text="Learn more" link="https://docs.microsoft.com/en-us/azure/container-apps/overview" newtab="true" >}}

## Tutorial

> Visit the [Azure docs](https://docs.microsoft.com/en-us/azure/container-apps/microservices-dapr) to try out a microservices tutorial, where you'll model two services and add Dapr:

Reviewer: ```suggestion
Visit the [Azure docs](https://docs.microsoft.com/en-us/azure/container-apps/microservices-dapr) to try out a microservices tutorial, where you'll deploy two Dapr-enabled services in Azure Container Apps:

```

---
type: docs
title: "Run Dapr in Azure Container Apps"
linkTitle: "Azure Container Apps"
weight: 3000

> description: "Learn how to run your Dapr apps on the Azure Container Apps serverless runtime"

Reviewer 2: ```suggestion
description: "Learn how to run your Dapr applications on the Azure Container Apps serverless platform "

```

---

> [Azure Container Apps](https://docs.microsoft.com/en-us/azure/container-apps/overview) is a serverless application centric hosting service where users do not see nor manage any underlying VMs, orchestrators, or other cloud infrastructure. Azure Container Apps enables executing application code packaged in any container and is unopinionated about runtime or programming model.

Reviewer 2: ```suggestion
[Azure Container Apps](https://docs.microsoft.com/en-us/azure/container-apps/overview) is a serverless application hosting service where users do not see nor manage any underlying VMs, orchestrators, or other cloud infrastructure. Azure Container Apps enables you to run your application code packaged in multiple containers and is unopinionated about runtimes or programming models that are used.
```

> Dapr is built-in as a first-class experience in Container Apps, letting you use Dapr building blocks without any manual deployment or management of Kubernetes or the Dapr runtime. Simply deploy your services and Dapr components and let the platform take care of the rest.

Reviewer 2: ```suggestion
Dapr is built-in to Container Apps, enabling you to use the Dapr API building blocks without any manual deployment of the Dapr runtime. You simply deploy your services with their Dapr components.

```

{{< button text="Learn more" link="https://docs.microsoft.com/en-us/azure/container-apps/overview" newtab="true" >}}

## Tutorial

> Visit the [Azure docs](https://docs.microsoft.com/en-us/azure/container-apps/microservices-dapr) to try out a microservices tutorial, where you'll deploy two Dapr-enabled services in Azure Container Apps:

Reviewer 2: ```suggestion
Visit the [Azure docs](https://docs.microsoft.com/en-us/azure/container-apps/microservices-dapr) to try out a microservices tutorial, where you'll deploy two Dapr-enabled applications to Azure Container Apps.

```
